sds_data_manager/utils/stackbuilder.py

```diff
@@ -194,23 +195,21 @@ def build_sds(
     )
 
     # This valid instrument list is from imap-data-access package
-    for instrument in imap_data_access.VALID_INSTRUMENTS:
-        ecr = ecr_construct.EcrConstruct(
-            scope=sdc_stack,
-            construct_id=f"{instrument}Ecr",
-            instrument_name=f"{instrument}",
-        )
-
-        batch_compute_resources.FargateBatchResources(
-            scope=sdc_stack,
-            construct_id=f"{instrument}BatchJob",
-            vpc=networking.vpc,
-            processing_step_name=instrument,
-            data_bucket=data_bucket.data_bucket,
-            repo=ecr.container_repo,
-            db_secret_name=db_secret_name,
-            efs_instance=efs_instance,
+    processing_volumes = [
+        batch.EfsVolume(
```

Comment:
Will this still be able to work in the same way with the example in imap_processing/examples/imap_processing/Dockerfile.efs and/or Dockerfile.processing?

Reply:
I'm not sure what you mean? It is working with the containers we have currently. We are mounting the EFS to `/mnt/spice` via the container_path below.